client/blocks/login/two-factor-authentication/waiting-notification-approval.jsx

```diff
@@ -62,7 +62,7 @@ class WaitingTwoFactorNotificationApproval extends Component {
 	componentWillReceiveProps( nextProps ) {
 		if ( ! this.props.pushSuccess && nextProps.pushSuccess ) {
 			const { translate } = this.props;
-			this.props.successNotice( translate( 'Verified successfully!' ) );
+			this.props.successNotice( translate( 'Logging in…' ) );
```

Comment:
Hi! I've found a possible matching string that has already been translated **32** times:
`translate( 'Logging In...' )` *ES Score: 16*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/change/2fa-success-copy)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).